app/views/idv/doc_auth/send_link.html.erb

```diff
@@ -8,6 +8,7 @@
 
 <h1 class='h3'>
   <%= t('doc_auth.headings.take_picture') %>
+</h1>
```

Comment:
😬 
yikes --- is there a way we could have caught this? Like a view spec?

Reply:
With the caveat that I'm not intimately familiar with the Ruby testing landscape, I could imagine a few approaches here:

- Integration tests where the contents of the heading are verified
   - e.g. `expect(page.find('h1')).to have_content(t('doc_auth.headings.take_picture'))`
   - Works because if the closing tag was missing, more content would be included
- Integration tests which validate the page's HTML (`page.html`) using Nokogiri
   - https://stackoverflow.com/a/24579476
- Integrate some sort of static analysis for ERB files
   - [better-html](https://github.com/Shopify/better-html)? [erb-lint](https://github.com/Shopify/erb-lint)? [rails-erb-lint](https://github.com/kathgironpe/rails-erb-lint)? These all seem more concerned with interpolation syntax validity than HTML validity.